ts/test/session/messages/ReceiptMessage_test.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import { beforeEach} from 'mocha';
+
+import { DeliveryReceiptMessage, ReadReceiptMessage } from '../../../session/messages/outgoing';
+import { SignalService } from '../../../protobuf';
+
+describe('ReceiptMessage', () => {
+    let readMessage: ReadReceiptMessage;
+    let deliveryMessage: ReadReceiptMessage;
+    let timestamps: Array<number>;
+
+    beforeEach(() => {
+        timestamps = [987654321, 123456789];
+        readMessage = new ReadReceiptMessage(Date.now(), '123456', timestamps);
+        deliveryMessage = new DeliveryReceiptMessage(Date.now(), '123456', timestamps);
+    });
+
+    it('content of a read receipt is correct', () => {
+        const plainText = readMessage.plainTextBuffer();
+        const decoded = SignalService.Content.toObject(SignalService.Content.decode(plainText));
+
+        expect(decoded.receiptMessage).to.have.property('type', 1);
+        expect(decoded.receiptMessage.timestamp).to.have.lengthOf(2);
+        expect(decoded.receiptMessage.timestamp[0]).to.have.property('low', timestamps[0]);
+        expect(decoded.receiptMessage.timestamp[1]).to.have.property('low', timestamps[1]);
```

Comment:
you can call `.toNumber()` on the timestamps to get the numerical values.

```
const timestamp = receiptMessage.timestamp[i].toNumber()
```

Reply:
Fixed